ftplugin/php/commands.vim

```diff
@@ -65,3 +65,7 @@ command! -buffer -nargs=0 PhpactorGenerateAccessors call phpactor#GenerateAccess
 ""
 " Automatically add any missing properties to a class
 command! -buffer -nargs=0 PhpactorTransform call phpactor#Transform()
```

Comment:
This text is used tl generate the docs, can be changed to : Copy the current class FQN (based on current filename)  to the clipboard